media-sound/teamspeak-server-bin/files/teamspeak.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=TeamSpeak3 Server
+After=network-online.target
+
+[Service]
+Type=simple
+User=teamspeak
+Group=teamspeak
+WorkingDirectory=/opt/teamspeak3-server
+Environment="LD_LIBRARY_PATH=/opt/teamspeak3-server"
+Environment="TS3SERVER_LICENSE=accept"
```

Comment:
Is this necessary when you already create the license-accepted file?

Reply:
No, it's not necessary, but I think, for the "just in case", it's nice to have it here. It doesn't hurt IMHO.

Reply:
Well, my main concern was that it's present in systemd unit but not in OpenRC init.d. But it's not a biggie.

Reply:
I've added an env var for the openrc script.